src/main/scala/org/marvin/executor/actions/PipelineAction.scala

```diff
@@ -60,7 +60,7 @@ class PipelineAction(metadata: EngineMetadata) extends Actor with ActorLogging{
 
         val futures:ListBuffer[Future[Done]] = ListBuffer[Future[Done]]()
 
-        for(artifactName <- engineActionMetadata.artifactsToLoad) {
+        for(artifactName <- engineActionMetadata.artifactsToPersist) {
```

Comment:
o objective here is to persist all artifacts and not only those that must be loaded...

artifactsToLoad brings all artifacts that must be loaded
artifactsToPersist brings all artifacts that must be persisted

Is a little bug .... very hiding....and hard to precept...